tsconfig.base.json

```diff
@@ -21,7 +21,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es2019"],
-    "baseUrl": "."
+    "lib": ["es2021"],
+    "baseUrl": ".",
+    "skipLibCheck": true
```

Comment:
@glasser this is maybe disappointing to turn on? `lru-cache@8` violates `exactOptionalPropertyTypes` so it triggers compilation errors. Thoughts?

Reply:
Has this been filed as an issue with them? Also looks like they just went to v9!

Reply:
I wouldn't consider it an issue necessarily, it's a configuration option that the project isn't opted in to that we are.